api/src/main/java/eu/darkbot/api/managers/NpcEventAPI.java

```diff
@@ -0,0 +1,24 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.API;
+
+/**
+ * Provides data for any Npc event
+ */
+public interface NpcEventAPI extends API.Singleton {
+
+    /**
+     * Get how long until the current {@link #getStatus()} changes.
+     * @return how long until the npc is active/inactive, in seconds
+     */
+    double getRemainingTime();
+    /**
```

Comment:
```suggestion

    /**
```
Spacing between methods